.circleci/config.yml

```diff
@@ -33,6 +33,22 @@ jobs:
       - store_test_results:
           path: build/reports/
 
+  xcode_9.1_swift_4.0.2:
+    macos:
+      xcode: "9.1.0"
```

Comment:
Why `9.1.0`? The forum post says to specify `9.1`: https://discuss.circleci.com/t/macos-build-image-update-xcode-9-1/17839

Reply:
Nevermind, I found this: https://discuss.circleci.com/t/xcode-9-1-release-for-circleci-2-0/17891